pepper-apis/src/main/resources/db-changes/schema/DDP-4737-tooltip.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="yufeng" id="20200612-tooltip-table">
+        <createTable tableName="tooltip">
+            <column name="tooltip_id" type="bigint" autoIncrement="true" startWith="1">
+                <constraints primaryKey="true" primaryKeyName="tooltip_pk"/>
+            </column>
+            <column name="text_template_id" type="bigint">
+                <constraints nullable="false"
+                             references="template(template_id)"
+                             foreignKeyName="tooltip_text_template_fk"/>
+            </column>
+        </createTable>
+    </changeSet>
+
```

Comment:
Do we need a new table for this? Could it just simply be a new column called tooltip_text_template_id in the tables that need it?

Reply:
I'm not sure if just a text template will suffice in representing tooltips. After a recent Slack convo about A-T's tooltip requirement, it seems like we don't need anything more. But I'm suspecting we might want other things related to tooltips down the line, so making it a table to give us room to grow. I can simplify this if folks feel that I'm overthinking this.

Reply:
I can't think of anything else that we would want. Some logic on when it is shown? Support for HTML? My suggestion is if you are not aware of specific very likely requirement that would necessitate additional properties, we should get rid of it. But it is your call.

Reply:
From what I can tell, other aspects related to tooltips are:
* Positioning, e.g. left, right, top, bottom, etc
* Timing, e.g. show after a delay of 1.5 secs, keep it on-screen for another second, etc
* Most recommend to keep tooltip as plain text, but perhaps we might want some "richer" text
* I think there are some where the tooltip shows a modal popup sort of thing, but I guess at that point it's probably not considered a "tooltip" anymore is it...

Arguably, some of these are presentation elements or browser/OS specifics, but it wouldn't be the first time we added presentation elements to the model... Also, in particular, I will like to hear from @fat23cat on implementation of tooltips on Angular side and if we need anything more (e.g. icon url).

Reply:
Will express hope we don't put any of the stuff on the server, except maybe "richer" text. Maybe. I don't think it is worth dedicating a new table to holding the one foreign key. But your call.

Reply:
Thanks for the feedback! Since this is the second time folks had questioned the design, I'll go ahead and simplify it so we have a new column instead of a new table, and change API to have just new string property instead of object.

Reply:
I'm with @MocanaAtBroad on keeping the presentation options on the front-end. As for making the item be a `string` instead of an `object`- it sounds good to me!